includes/admin/settings/register-settings.php

```diff
@@ -979,10 +979,6 @@ function edd_settings_sanitize( $input = array() ) {
 						unset( $output[ $key ] );
 					}
 					break;
-				case 'select':
-					// Ensures that Multiple Selects that have been cleared out save as Empty
-					if ( empty( $_POST['edd_settings'][ $key ] ) ) $output[ $key ] = '';
-					break;
```

Comment:
@cklosowski With this bit of code removed, if a value is saved into a Multiple Select and then the User attempts to clear out all of its values and Save those changes it will not work.

I see the problem that you mentioned in #4736 since this _is_ a fairly heavy-handed approach and apparently effects Select Fields not on the same Page. However, the inability to clear out the value isn't very good :/

We may need to find another way.